docs/sphinx_api_docs_source/build_sphinx_api_docs.py

```diff
@@ -279,15 +279,14 @@ def _parse_and_process_html_to_mdx(  # noqa: C901
             else:
                 link_sidebar_entry = self._get_sidebar_entry(html_file_path=href_path)
 
-            relative_link = self._relative_path_between_documents(
-                sidebar_entry.mdx_relpath, link_sidebar_entry.mdx_relpath
-            )
+            relative_link = link_sidebar_entry.mdx_relpath
+
             without_extension = str(relative_link).replace(".mdx", "")
```

Comment:
Is this still necessary?

Reply:
yes, because that relative link still have the ".mdx" extension on them